### PR TITLE
Clean up some more disposable usage

### PR DESCRIPTION
**Pull request by anthonydresser**

### CODE REVIEW

File: src/sql/workbench/parts/query/browser/queryActions.ts [hunk 1]
```diff
@@ -605,11 +604,11 @@ export class ListDatabasesActionItem implements IActionViewItem {
 				actionLabel: nls.localize('listDatabases.toggleDatabaseNameDropdown', "Select Database Toggle Dropdown")
 			});
 			this._dropdown.onValueChange(s => this.databaseSelected(s));
-			this._toDispose.push(this._dropdown.onFocus(() => this.onDropdownFocus()));
+			this._register(this._dropdown.onFocus(() => this.onDropdownFocus()));
```
Comment (Charles-Gagnon) [inline src/sql/workbench/parts/query/browser/queryActions.ts]: This should probably be added to the registration as well

**Comment by Charles-Gagnon** [comment]
> 			this._databaseSelectBox.disable();

This should probably be added to the registration as well

---
Refers to: src/sql/workbench/parts/query/browser/queryActions.ts:597 in 07ee053. [](commit_id = 07ee053b75153bf4d3e8f459d712282a9b89cbe8, deletion_comment = False)

File: src/sql/workbench/parts/query/browser/queryActions.ts (hunk 1 above)
Comment (anthonydresser) [reply src/sql/workbench/parts/query/browser/queryActions.ts]: What do you mean add to the registration?

Comment (Charles-Gagnon) [reply src/sql/workbench/parts/query/browser/queryActions.ts]: Guess this got put on the wrong line : 

```
this._dropdown.onValueChange(s => this.databaseSelected(s));
```

to

```
this._register(this._dropdown.onValueChange(s => this.databaseSelected(s)));
```

Comment (anthonydresser) [reply src/sql/workbench/parts/query/browser/queryActions.ts]: O i see

**Pull request merged**